action/report/textreporter.go

```diff
@@ -81,19 +81,32 @@ func (tr *TextReporter) getSLOStrText(i int, upper bool) (string, error) {
 	return str, nil
 }
 
+func (tr *TextReporter) printVersions(w *tabwriter.Writer) {
```

Comment:
Please incorporate similar logic for HTML reporter (for example, using a new method defined on the reporter).

Reply:
done